.github/workflows/author_approval.yml

```diff
@@ -0,0 +1,43 @@
+name: Author Approval Label
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  label:
+    permissions:
+        pull-requests: write
+    runs-on: ubuntu-latest
+    # Run on comments that are:
+    # 1) on PRs, not issues,
+    # 2) not from bot users and,
+    # 3) include the string "merge approved"
+    # If so, we will do the work to check that the commenter is the package author,
+    # and conditionally apply the author-approved label.
+    # note: `merge approved` here is NOT case-sensitive, see https://docs.github.com/en/actions/learn-github-actions/expressions#contains
+    if: ${{ github.event.issue.pull_request && github.event.issue.user.type != 'Bot' && contains(github.event.comment.body, 'merge approved') }}
```

Comment:
```suggestion
    if: github.event.issue.pull_request && (github.event.issue.user.type != 'Bot') && (github.event.comment.body == '[merge approved]') && (github.event.comment.body == '[noblock]')
```

Reply:
This looks much more complicated, and I’m sure users will make mistakes. I’d much rather what I wrote and to change the RegistryCI check to not block on this phrase (which is already done in my PR)

Reply:
I don't love the special-casing in https://github.com/JuliaRegistries/RegistryCI.jl/pull/539.

It also seems simpler to me to tell users "always include `[noblock] in your comments", instead of saying "sometimes you need `[noblock]` in your comments, and other times you don't".

Can we just add text to the AutoMerge comment of the following form?

> If you are the author of this package, and you want to approve this PR and bypass the sequential-version-number check, please post a comment on this PR with the following text verbatim: `[merge approved] [noblock]`

Reply:
I think a longer comment means more chance to miss that text along with all the rest of it. It's already so long!

As a compromise, what about `[merge approved]` (with the square brackets), but without requiring `[noblock]`? Then the rule is "commands to AutoMerge occur in square brackets and comments with commands do not block automerge".

Reply:
I think I can live with that.

Reply:
```suggestion
    if: ${{ github.event.issue.pull_request && github.event.issue.user.type != 'Bot' && contains(github.event.comment.body, '[merge approved]') }}
```